src/app/common/KFilterBar.vue

```diff
@@ -13,15 +13,14 @@
     >
       <span class="visually-hidden">Focus filter</span>
 
-      <KIcon
-        aria-hidden="true"
-        class="k-filter-icon"
-        :color="KUI_COLOR_TEXT_NEUTRAL_WEAK"
-        data-testid="k-filter-bar-filter-icon"
-        hide-title
-        icon="filter"
-        :size="KUI_ICON_SIZE_30"
-      />
+      <span class="k-filter-icon">
+        <FilterIcon
+          aria-hidden="true"
```

Comment:
suggestion: you can use the `decorative` prop here instead of manually setting `aria-hidden`

This suggestion applies to all icons where you have the `aria-hidden` attribute in place

Reply:
I’ve replaced the three instances, thanks.